test/basic-images.test

```diff
@@ -214,6 +214,18 @@ test_expect_success "hdimage2" "
 	test_must_fail run_genimage hdimage2.config test.hdimage
 "
 
+test_expect_success fdisk,sfdisk "hdimage4" "
```

Comment:
This fails with the old test setup, right? We should skip the test for old versions of `fdisk` / `sfdisk`.
Untested:
```
sfdisk -h | grep -q gpt && test_set_prereq sfdisk-gpt
fdisk -h | grep -q gpt && test_set_prereq fdisk-gpt
test_expect_success fdisk-gpt,sfdisk-gpt "hdimage4" "
...
```